policyengine_canada/parameters/gov/cra/tax/income/credits/climate_action_incentive/age_max_child.yaml

```
@@ -0,0 +1,10 @@
+description: Age after which an individual is no longer considered a child.
```

I'd suggest calling this `adult_age` instead.
```suggestion
description: Age at which the Climate Action Incentive considers a person an adult.
```